src/components/RowItem.tsx

```diff
@@ -34,14 +35,14 @@ function RowItem<T>(props: Props<T>) {
     drag(itemKey);
   }, []);
 
-  const { renderItem, item, itemKey, extraData } = props;
+  const { renderItem, item, itemKey, extraData, passIndexToRenderItem } = props;
   return (
     <MemoizedInner
       isActive={activeKey === itemKey}
       drag={drag}
       renderItem={renderItem}
       item={item}
-      index={keyToIndexRef.current.get(itemKey)}
+      index={passIndexToRenderItem? keyToIndexRef.current.get(itemKey): undefined}
```

Comment:
Tiny nit - this ternary doesn't seem to be in the correct format. Can you make sure to format this file with Prettier before submitting?